custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,159 @@
-                        server_name
-                        not in dict_get(
-                            self._state, "previous_state.telemetry.openvpn.servers", {}
-                        ).keys()
-                    ):
+                    new_property = f"{prop_name}_{label}"
+                    interface[new_property] = int(round(value, 0))
+
+            for server_name in dict_get(self._state, "telemetry.openvpn.servers", {}):
+
+                if server_name not in dict_get(
+                    self._state, "previous_state.telemetry.openvpn.servers", {}
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)